mne/io/edf/edf.py

```diff
@@ -576,7 +576,7 @@ def _read_edf_header(fname, exclude):
         for i, unit in enumerate(units):
             if i in exclude:
                 continue
-            if unit == 'uV':
+            if unit in ('uV', 'µV'):
```

Comment:
IDK if this is unnecessarily conservative, or on the contrary, totally pointless. I *think* it might be pointless because if it wasn't needed before, then EDF headers will never say "µV" anyway.

Reply:
Agreed we shouldn't need to modify internal values until we find files that use these new chars